app/jenkins-for-jira-ui/src/components/JenkinsServerList/JenkinsServerList.tsx

```diff
@@ -56,29 +65,44 @@ const JenkinsServerList = (): JSX.Element => {
 		</ButtonGroup>
 	);
 
-	return jenkinsServers?.length ? (
-		<>
-			<div className={headerContainer}>
-				<PageHeader actions={pageHeaderActions}>Jenkins configuration</PageHeader>
-			</div>
+	let contentToRender;
```

Comment:
eslint was bitching about nesting ternaries so had to extract the content we want to render so I we can handle all 3 scenarios.